backend/db_migrations/000019_duplicate-mealplan-name-input.down.sql

```diff
@@ -0,0 +1,2 @@
+GRANT execute on function app.duplicate_meal_plan(bigint, bigint) to app_admin, app_meal_designer;
+drop function if exists app.duplicate_meal_plan(mealplan_id bigint, p_id bigint);
```

Comment:
Should not drop the complete function. it should be the previous function from the previous migration script.